SWLOR.Game.Server/Item/RepairKit.cs

```diff
@@ -36,12 +36,11 @@ public void ApplyEffects(NWCreature user, NWItem item, NWObject target, Location
             int repairAmount = tech * 2;
 
             if (skillType == SkillType.Armorsmith)
-                repairAmount += item.CraftBonusArmorsmith;
+                repairAmount += item.CraftBonusArmorsmith + (_perk.GetPCPerkLevel(user.Object,PerkType.ArmorRepair) * 5);
             else if (skillType == SkillType.Weaponsmith)
-                repairAmount += item.CraftBonusWeaponsmith;
+                repairAmount += item.CraftBonusWeaponsmith + (_perk.GetPCPerkLevel(user.Object,PerkType.WeaponRepair) * 5);
             else if (skillType == SkillType.Engineering)
-                repairAmount += item.CraftBonusEngineering;
-
+                repairAmount += item.CraftBonusEngineering + (_perk.GetPCPerkLevel(user.Object,PerkType.ElectronicRepair) * 5);
```

Comment:
My first instinct is that these bonuses are way too high. At max, that's I think:

4 * 2 = 8 (tech bonus)
Item bonus (an unknown value between 0-20+)
4 * 5 = 20 (perk bonus)

That's a lot of points! I would feel more comfortable if we started small with 2 and if we need to boost the benefit more, we can. It's just hard to go backwards after we've set the value.